appmesh-preview/sdk/api.json

```diff
@@ -1633,8 +1633,7 @@
     "GatewayRouteHostnameRewrite":{
       "type":"structure",
       "members":{
-        "defaultTargetHostname":{"shape":"DefaultGatewayRouteRewrite"},
-        "targetBased":{"shape":"GatewayRouteTargetBasedHostnameRewrite"}
```

Comment:
why'd this get removed?

Reply:
Yes. this is intentional. This was an experimental property that got added before Rewrite release. This is no longer supported.